osu.Game/Screens/Import/FileImportScreen.cs

```diff
@@ -0,0 +1,320 @@
+                                                    RelativeSizeAxes = Axes.Both,
+                                                    Child = currentFileText = new TextFlowContainer(t => t.Font = OsuFont.Default.With(size: 30))
+                                                    {
+                                                        AutoSizeAxes = Axes.Y,
+                                                        RelativeSizeAxes = Axes.X,
+                                                        Anchor = Anchor.Centre,
+                                                        Origin = Anchor.Centre,
+                                                        TextAnchor = Anchor.Centre
+                                                    },
+                                                },
+                                            }
+                                        },
+                                    },
+                                    new Drawable[]
+                                    {
+                                        new Container
+                                        {
+                                            RelativeSizeAxes = Axes.X,
+                                            AutoSizeAxes = Axes.Y,
+                                            Children = new Drawable[]
+                                            {
+                                                new Box
+                                                {
+                                                    Colour = overlayColourProvider.Background4,
+                                                    RelativeSizeAxes = Axes.Both
+                                                },
+                                                new FillFlowContainer
+                                                {
+                                                    RelativeSizeAxes = Axes.X,
+                                                    AutoSizeAxes = Axes.Y,
+                                                    Spacing = new Vector2(10),
+                                                    Children = new Drawable[]
+                                                    {
+                                                        new SettingsEnumDropdown<FileFilterType>
+                                                        {
+                                                            Anchor = Anchor.BottomCentre,
+                                                            Origin = Anchor.BottomCentre,
+                                                            LabelText = "File Type",
+                                                            Current = filterType,
+                                                            Margin = new MarginPadding { Bottom = 15 }
+                                                        },
+                                                        new GridContainer
```

Comment:
as an example of restructuring, this should not require a `GridContainer` at all.